src/components/enhanced-table/enum-filter/EnumFilter.tsx

```diff
@@ -21,7 +42,8 @@ const EnumFilter = ({ column, filter, setFilter, clearFilter }) => {
 
   const endAdornment = (
     <IconButton
-      className={filter.length ? 'visible' : 'hidden'}
+      aria-hidden={isFilterEmpty}
+      className={!isFilterEmpty ? 'visible' : 'hidden'}
```

Comment:
```suggestion
      className={isFilterEmpty ? 'hidden' : 'visible'}
```
I think it would be better and more readable. But it is up to you)